waspc/data/Generator/templates/Dockerfile

```diff
@@ -55,15 +55,15 @@ WORKDIR /app
 # Copying the top level 'node_modules' because it contains the Prisma packages
 # necessary for migrating the database.
 COPY --from=server-builder /app/node_modules ./node_modules
-# Copying the SDK because 'validate-env.mjs' executes independent of the bundle
+# Copying the SDK because the server bundle doesn't bundle the SDK
 # and references the 'wasp' package.
 COPY --from=server-builder /app/.wasp/out/sdk .wasp/out/sdk
-# Copying 'server/node_modules' because 'validate-env.mjs' executes independent
-# of the bundle and references the dotenv package.
+# Copying 'server/node_modules' because we require dotenv package to
+# load environment variables
+# TODO: replace dotenv with native Node.js environment variable loading
```

Comment:
It's always nicer to use natively supported methods than a package to do the same thing. It's available since [Node.js 20](https://nodejs.org/en/blog/release/v20.6.0) and we should probably go for it when Node.js 20 becomes our minimum version.